vhost_rs/src/vhost_user/master_req_handler.rs

```diff
@@ -11,33 +11,37 @@ use std::sync::{Arc, Mutex};
 
 use super::connection::Endpoint;
 use super::message::*;
-use super::{Error, HandlerResult, Result};
+use super::{Error, Result};
+
+fn error_code<T>(err: std::io::Error) -> Result<T> {
+    Err(Error::ReqHandlerError(err))
+}
 
 /// Trait to handle vhost-user requests from the slave to the master.
 pub trait VhostUserMasterReqHandler {
     // fn handle_iotlb_msg(&mut self, iotlb: VhostUserIotlb);
     // fn handle_vring_host_notifier(&mut self, area: VhostUserVringArea, fd: RawFd);
 
     /// Handle device configuration change notifications from the slave.
-    fn handle_config_change(&mut self) -> HandlerResult<()> {
-        Err(std::io::Error::from_raw_os_error(libc::ENOSYS))
+    fn handle_config_change(&mut self) -> Result<()> {
+        error_code(std::io::Error::from_raw_os_error(libc::ENOSYS))
```

Comment:
I made the above error conversion because `SlaveFsCacheReq` needs to errors from Endpoint and I prefer not to losing that error information by mapping it to raw os errors.
(I'll also clarify it in the commit message).